napari/layers/utils/layer_utils.py

```diff
@@ -228,6 +228,14 @@ def calc_data_range(data, rgb=False) -> Tuple[float, float]:
     if data.dtype == np.uint8:
         return (0, 255)
 
+    if isinstance(data, np.ndarray) and data.ndim < 3:
+        min_val = _nanmin(data)
+        max_val = _nanmax(data)
+        if min_val == max_val:
+            min_val = 0
+            max_val = 1
+        return float(min_val), float(max_val)
```

Comment:
```suggestion
        return (float(min_val), float(max_val))
```
This should be a tuple of float, right?

Reply:
those two version are equivalent :)

Reply:
for some mindblow, try:
```py
a = 1,
print(a)
```

Reply:
I mean, I like that 😄 
But I'm shocked that the type checkers and everything are all fine with that and not having a cow.
Clearly I have yet to understand typing in Python.

Reply:
Well this is not a typing thing, it's a syntax thing. The type is tuple, even though there's more than one way to write a tuple.